auth/credentials/downscope/downscope.go

```diff
@@ -51,6 +58,15 @@ func (c Options) client() *http.Client {
 	return internal.CloneDefaultClient()
 }
 
+// identityBindingEndpoint returns the identity binding endpoint with the
+// configured universe domain.
+func (c Options) identityBindingEndpoint() string {
```

Comment:
nit: would you mind changing this method and the one above to pointer semantics.

Reply:
Done, good catch!